p2p/module_raintree_test.go

```diff
@@ -64,7 +113,64 @@ func TestRainTreeNetworkCompleteThreeNodes(t *testing.T) {
 		validatorId(2): {1, 0}, // val_2 does a single network read (from val_1)
 		validatorId(3): {1, 0}, // val_2 does a single network read (from val_3)
 	}
-	testRainTreeCalls(t, originatorNode, expectedCalls)
+	testRainTreeCalls(t, originatorNode, expectedCalls, false, false)
+}
+
+func TestRainTreeNetworkCompleteThreeNodesRedundancyLayerEnabled(t *testing.T) {
+	// 	          val_1
+	// 	   ┌───────┴────┬─────────┐
+	//   val_2        val_1     val_3
```

Comment:
Might be a github thing but need to fix spacing if not.

<img width="428" alt="Screenshot 2022-11-21 at 7 54 07 PM" src="https://user-images.githubusercontent.com/1892194/203217340-b3f5fab1-9cb8-4d01-b01b-5ca653522100.png">